### PR TITLE
rewrote the background games demo to use the new run_background_game function from pelita.utils

### PR DESCRIPTION
**Pull request by otizonaizit**

### CODE REVIEW

**Comment by otizonaizit** [comment]
Sorry, to run this you need pelita https://github.com/ASPP/pelita/pull/674

File: demo10_background_games.py [hunk 1]
```diff
@@ -16,36 +15,69 @@
...
-                                      team_names=team_names)
-
-    # after the game is finished, update the stats
-    if game_state['whowins'] == defenders_index:
-        statistics['defender_wins'] += 1
-    elif game_state['whowins'] == attackers_index:
-        statistics['attacker_wins'] += 1
+
+    # dictionary to store game parameters
+    game = {}
```
Comment (Debilski) [inline demo10_background_games.py]: I think having both `game` and `game_state` is very confusing.

Comment (otizonaizit) [reply demo10_background_games.py]: done

**Comment by otizonaizit** [comment]
I added an example on how to replay a game. This PR should be merged soon.

**Pull request merged**